client/src/app/users/user-list.component.scss

```diff
@@ -5,6 +5,7 @@
 
 .user-card {
   margin-bottom: 10px;
+  flex: 1 1 280px;
```

Comment:
I think documenting (for other developers and for the students) what this is doing would be helpful.